src/hooks/useEditorState.ts

```diff
@@ -7,7 +7,7 @@ import { EditorContext } from "../contexts/EditorContext.js";
  * Provides access to the current EditorState value.
  */
 export function useEditorState(): EditorState | null {
-  const { editorState } = useContext(EditorContext);
+  const { state: editorState } = useContext(EditorContext);
```

Comment:
Would you mind if I pulled the change in https://github.com/nytimes/react-prosemirror/pull/84 here too? I'm realizing we've relied on non-nullability in a bunch of places now, and want to remove that from the diffs in getting an understanding of the impact of this branch on some of our code.

Reply:
Turns out this was already non-nullable everywhere, this hook was just incorrectly typed! I just pushed the update